--- src/Inventor/Qt/Makefile.in	Tue Feb  8 16:33:27 2000
+++ src/Inventor/Qt/Makefile.in.corrected	Tue Feb  8 16:32:34 2000
@@ -218,7 +218,8 @@
 HEADERS =  $(libSoQtinc_HEADERS)
 
 DIST_COMMON =  $(libSoQtinc_HEADERS) Makefile.am Makefile.in \
-SoQtRenderArea.cpp.in SoQtRenderArea.h.in
+$(top_srcdir)/src/Inventor/Qt/common/SoAnyRenderArea.cpp.in \
+$(top_srcdir)/src/Inventor/Qt/common/SoAnyRenderArea.h.in
 
 
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
@@ -249,9 +250,9 @@
 	cd $(top_builddir) \
 	  && CONFIG_FILES=$(subdir)/$@ CONFIG_HEADERS= $(SHELL) ./config.status
 
-SoQtRenderArea.h: $(top_builddir)/config.status SoQtRenderArea.h.in
+SoQtRenderArea.h: $(top_builddir)/config.status $(top_srcdir)/src/Inventor/Qt/common/SoAnyRenderArea.h.in
 	cd $(top_builddir) && CONFIG_FILES=$(subdir)/$@ CONFIG_HEADERS= $(SHELL) ./config.status
-SoQtRenderArea.cpp: $(top_builddir)/config.status SoQtRenderArea.cpp.in
+SoQtRenderArea.cpp: $(top_builddir)/config.status $(top_srcdir)/src/Inventor/Qt/common/SoAnyRenderArea.cpp.in
 	cd $(top_builddir) && CONFIG_FILES=$(subdir)/$@ CONFIG_HEADERS= $(SHELL) ./config.status
 
 mostlyclean-libLTLIBRARIES:
