Index: configure
===================================================================
RCS file: /export/cvsroot/SoQt/configure,v
retrieving revision 1.46
diff -u -r1.46 configure
--- configure	2000/06/05 13:49:33	1.46
+++ configure	2000/06/08 09:57:37
@@ -6149,7 +6149,7 @@
     (echo ':t
 /@[a-zA-Z_][a-zA-Z_0-9]*@/!b' && cat $ac_cs_root.sfrag) >$ac_cs_root.s$ac_sed_frag
     if test -z "$ac_sed_cmds"; then
-      ac_sed_cmds="sed -f $ac_cs_root.s$ac_sed_frag"
+      ac_sed_cmds="tr -d \"
\" | sed -f $ac_cs_root.s$ac_sed_frag"
     else
       ac_sed_cmds="$ac_sed_cmds | sed -f $ac_cs_root.s$ac_sed_frag"
     fi
@@ -6159,7 +6159,7 @@
   fi
 done
 if test -z "$ac_sed_cmds"; then
-  ac_sed_cmds=cat
+  ac_sed_cmds="tr -d \"
\""
 fi
 
 EOF
